### ISSUE TITLE
AS Drums: AS RCK HiHAT

### ISSUE THREAD

**Issue by Cordoha** [AScustomWorks/AS#24]
Doesn't display in Linux. Might be the same issue as before with the name of the SVG I think it was? Please have a look at your other modules as well to see if there are any other issues. I just bought them, but not really satisfied at the moment. Especially since they're so quiet and need more voltage.

Also, please make all the modules in the Drums output to 16V, or similar to Vult's voltage. I have to use the waveshaper plugin to increase the voltage to have any competency with Vult modules.

Other than that, they're okay so far!
Thanks.

**Comment by AScustomWorks** [comment]
Hi Justin, thanks a lot for reporting this issue.
Indeed, the module is looking for HihatsSDS.svg, but the correct name should be HihatsSdS.svg. Sorry for the trouble, I'm fixing that and making a new release asap (please rename the file by hand if you can, while the new version is available).

About the audio levels, I'll review them and try to rise the overall level a bit where possible, but consider this:

The max output level setting on each module has some headroom to avoid output saturation. While you have the GAIN knob to increase the output, even when maxed out it won't won't get to the point of saturation because of the way the design went thru - quick and easy build blocks for drum kits -.

So, by thinking of each module as a part of a drum kit, the output level on each module was carefully set to be a close match with the rest of the modules to avoid as much as possible any extra volume tweaking afterwards. This increases the headroom of course, one little sacrifice that makes the initial setup easier. 

Besides that, the best way to get a good sound balance on a music mix is to reduce output levels on separated channels or mix groups as needed, not the other way around. That being said, I don't think there's a need to compete against the levels of any other sound source.

**Comment by Cordoha** [comment]
Right, makes sense, thanks for the reply. 

I just found quite the difference in levels between the modules I was using, almost like they the sound was non-existent compared to the modules. With the Mixer as high as I could get it, it wasn't loud enough. With the difference, I had to use the Waveshaper at 16V, and turned it up enough to make it sound accordingly, or to the best I could(to make it sound proper without saturation or noise saturation.

I was using Lateralus, Resonator, and a couple others. Vult's modules definitely have a loudness, so it could have been that. I can't really blame it on my equipment either since I use an interface with studio monitors, studio subwoofer. :P

Anyway, thanks a lot for the reply, and the great modules.
Cheers.
